### PR TITLE
add 1 day timeout to sync scripts

### LINKED ISSUE #531: Add timeout to mirror healthcheck script
Sometimes healthchecks freeze up (usually because of upstream network problems). Since they use systemd timers instead of cronjobs, this means future healthchecks won't run.

Luckily this triggers our alerting, so someone can go into fallingrocks and manually kill the healthcheck. But this should happen automatically.

We could set the timeout inside the Python script, but I think it would be better to hangle it within systemd. [This thread](https://unix.stackexchange.com/questions/227017/how-to-change-systemd-service-timeout-value) could be useful.

### PR DESCRIPTION
**Pull request by dkess**
Fixes #531.

### CODE REVIEW

File: modules/ocf_mirrors/templates/systemd/sync.service.erb [hunk 1]
```diff
@@ -8,3 +8,4 @@ Type=simple
 Environment="<%= env %>=<%= val %>"
 <% end -%>
 ExecStart=<%= @exec_start %>
+TimeoutStartSec=86400 # one day should be good enough for anyone
```
Comment (cg505) [inline modules/ocf_mirrors/templates/systemd/sync.service.erb]: My understanding is that comments need to be on their own lines to be syntactically correct. Just put it above.

Comment (keur) [reply modules/ocf_mirrors/templates/systemd/sync.service.erb]: nice. you are correct `system.syntax(7)`.

**Comment by jvperrin** [comment]
(note that this is changing to having a timeout for both sync scripts and for the healthcheck scripts since they both use the same timers)

**Comment by boomtow** [comment]
How was that did it work

**Pull request merged**